Add download statistics and zenodo link scripts

marabuuu opened the pull request:
Hi all,

This PR adds two scripts: one to summarise download statistics and one to store zenodo links from DOI. Hereby, the corresponding Issue numbers can be found in individual commits.

closes #79 
closes #81 

Best,
Mara

haesleinhuepf reviewed (changes requested):
Hi @marabuuu ,

as just discussed:
* Let's remove the call of the new script from the .github/workflow/book.yml file
* Add a date, e.g. 20240619 to the filename of the CSV file that's created.

This will allow us to have CSV files for different points in time and later we can plot how downloads developed over the zears :-)

Thanks!

marabuuu commented:
Hi @haesleinhuepf,

thank you for your help. I have incorporated the changes. How can I let GitHub know that I addressed the requested changes?

Best,
Mara

haesleinhuepf commented:
Thank you @marabuuu ! 

> How can I let GitHub know that I addressed the requested changes?

If I _had used_ this syntax:
```
* [ ] task 1
* [ ] task 2
```

You could tick the boxes:

* [ ] task 1
* [ ] task 2

Thanks again!
